.env.example

```
@@ -9,3 +9,31 @@ DB_PASS=your_db_password
 DB_SYNC=false
 
+# E2E Test Database (NEVER point these at production/dev DBs)
+TEST_DB_HOST=localhost
+TEST_DB_PORT=5432
+TEST_DB_USER=postgres
+TEST_DB_PASS=your_db_password
+TEST_DB_NAME=emart_test
+
+# JWT
+JWT_SECRET=your_jwt_secret
+JWT_EXPIRES_IN=1d
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_EXPIRES_IN key should go before the JWT_SECRET key

(UnorderedKey)